repos/annotation-studio/src/Controller/CaptureModelController.php

```diff
@@ -112,6 +112,45 @@ public function captureModelAction()
         );
     }
 
+    public function translationAction() {
+        $site = $this->currentSite();
+        if ($site) {
+            $file = __DIR__ . '/../../../../translations/s/' . $site->slug() . '/annotation-studio.json';
+            if (file_exists($file)) {
```

Comment:
Should be `is_file($file)`. `file_exists(...)` will return `true` for directories too.

Reply:
Ah good catch!